src/components/KDropdown/KDropdownItem.vue

```diff
@@ -24,11 +24,17 @@
   </li>
 </template>
 
-<script lang="ts" setup>
+<script lang="ts">
 import type { DropdownItem, DropdownItemRenderedRecord, DropdownItemRenderedType, DropdownItemType } from '@/types'
 import type { PropType } from 'vue'
 import { computed, useAttrs } from 'vue'
 
+export default {
```

Comment:
suggestion: if you need this you can use defineOptions

Reply:
Done